apps/src/applab/designElements/label.jsx

```diff
@@ -155,8 +156,15 @@ module.exports = {
 
     var padding = parseInt(element.style.padding, 10);
 
+    //Below line allows us to wrap text in a reasonable manner. Set a max width that is at least a third but no more
+    //than the full width of the screen. That forces text the label to stop growing at a certain point
+    var maxWidthInt = element.style.left ? Math.max(Applab.appWidth - parseInt(element.style.left), Applab.appWidth / 3)
+        : Applab.appWidth;
+
+    element.style.maxWidth = maxWidthInt + 'px';
```

Comment:
Can you please make sure the following works:
1. create a text label and type in it until it reaches the right edge and wraps
2. drag it away from the right edge of the screen
3. drag the text area to expand it to the right --> The text should grow to fill the new width.
It looks like it should be working because we set the maxWidth in the `resize` handler, but just double-checking. sorry I missed this in the screencast.